Finjector.Web/ClientApp/src/pages/Entry.tsx

```diff
@@ -171,15 +176,19 @@ const Entry = () => {
             updateName={(n) => setSavedChart((c) => ({ ...c, name: n }))}
           />
         </div>
+
         <div className="row mb-5">
           <div className="col-md-6">
-            <div className="form-text">
-              Current Team: {savedChartQuery.data?.chart.teamName}
-              <br />
-              Current Folder: {savedChartQuery.data?.chart.folder?.name}
-            </div>
+            {savedChartQuery.data && (
+              <div className="form-text">
+                Current Team: {savedChartQuery.data?.chart.teamName}
```

Comment:
why check for null on `savedChartQuery.data?` here when you just checked 2 lines above that it wouldn't be null?

Reply:
Fixing, but the why is that it was already there and I added the line 182 and didn't look closer